src/auth/providers.js

```diff
@@ -267,6 +272,7 @@ function getOAuthLoginURL(auth, providerName, redirectUrl) {
 /** @namespace */
 function googleProvider(auth) {
   const loginRoute = auth.isAppClient() ? 'providers/oauth2-google/login' : 'providers/oauth2-google/login';
+  const providerType = 'google';
```

Comment:
we should declare these special strings in constants up top. we use em all over (including in tests)